nautobot/dcim/migrations/0030_migrate_region_and_site_data_to_locations.py

```diff
@@ -61,11 +68,21 @@ def create_region_location_type_locations(region_class, location_class, region_l
                 f"{e.args[0]} \nPlease consider changing the slug attribute of this Region instance to resolve this conflict."
             )
 
+    region_lt_locations = location_class.objects.filter(location_type=region_lt)
+    regions = region_class.objects.all()
+    # Cache the REGION_TO_LOCATION_LOOKUP for later use
+    for region in regions:
+        region.migrated_location = region_lt_locations.get(name=region.name)
+    region_class.objects.bulk_update(regions, ["migrated_location"], 1000)
+
 
 def create_site_location_type_locations(
     site_class,
     location_type_class,
     location_class,
+    tagged_item_class,
+    location_ct,
+    site_ct,
```

Comment:
Is it safe to pass in `apps` and use `apps.get_model` for the model classes? I assume this stays in Python and doesn't incur an extra database hit like the ContentTypes do

Reply:
so the `apps` argument is an instance of `django.apps.registry.Apps` as explained by the Django [Documentation](https://docs.djangoproject.com/en/4.1/ref/migration-operations/#runpython).
And the `get_model()` method does not incur database hits because it looks like it is getting its information from `self.app_configs` which is an in-memory store of `OrderDict()`, prepopulated by going through `INSTALLED_APPS`. https://docs.djangoproject.com/en/2.1/_modules/django/apps/registry/

Reply:
Not blocking but if you wanted to clean up the signature for this method you could pass in `apps` instead of the `site_class`, `location_type_class`, `location_class` and `tagged_item_class` arguments